src/gennylib/include/gennylib/Node.hpp

```diff
@@ -21,6 +21,7 @@
 #include <utility>
 #include <variant>
 #include <vector>
+#include <typeinfo>
```

Comment:
Is this needed?

Reply:
Ah remnant from my hack to detect STLs at runtime and to special logic lol, good catch